sklearn/feature_selection/univariate_selection.py

```diff
@@ -17,7 +17,7 @@
                      safe_mask)
 from ..utils.extmath import norm, safe_sparse_dot, row_norms
 from ..utils.validation import check_is_fitted
-from .base import SelectorMixin
+from .base import SelectorMixin, wrapper_scorer
```

Comment:
What are you doing in this file? I think you're making this PR far too big, and it's diverging in scope.